src/site/markdown/execution-and-routing.md

```diff
@@ -123,7 +123,7 @@ The generic endpoint proxy sends the request to the service:
     Accept: */*
     User-Agent: whatever
     Apix-Ldp-Resource: http://fedora01.backend.local/repository/foo/
-    Apix-Exposed-Uri: http://archive.example.org/services/foo/svc:Thumbnail/
+    Apix-Exposed-Uri: http://archive.example.org/services/foo/svc:Thumbnail/small
```

Comment:
Actually, the former is correct.   `Apix-Exposed-Uri` is just the exposed _endpoint_, so it'll never change.  The use case in the text is "a service may use this as a base URI when formulating a response that contains hypertext or links".  So if a service generates links to resources within itself (e.g. `small`, or `a/b/c/d`, it can represent those links as relative URIs as -is, or use `Apix-Exposed-Uri` as a baseuri to construct absolute URIs like `http://archive.example.org/services/foo/svc:Thumbnail/a/b/c/d`.  

As an example, suppose a service generates RDF as N-triples.  The N-triples format requires absolute URIs, so they'd need to be constructed as `${Apix-Exposed-Uri}/relative/uri` before serializing.